scripts/tutorials/wdl/README.md

```diff
@@ -0,0 +1,31 @@
+### Easily find information in each tutorial
+---
+
+The WDL scripts in this folder are designed to teach you aspects and features of WDL, using GATK as it's main tool. They follow the tutorial documents located on the WDL website, under the Tutorials category
```

Comment:
but actually would be better phrased as "mainly using GATK tools for demo purposes"